WordPress/Classes/Categories/NSDate+StringFormatting.m

```diff
@@ -1,8 +1,22 @@
 #import "NSDate+StringFormatting.h"
 #import "WordPress-Swift.h"
+#import <FormatterKit/TTTTimeIntervalFormatter.h>
 
 @implementation NSDate (StringFormatting)
 
++ (NSDateFormatter *)shortDateFormatter
+{
+    static NSDateFormatter *_shortDateFormatter = nil;
+    static dispatch_once_t _onceToken;
+    dispatch_once(&_onceToken, ^{
+        _shortDateFormatter = [[NSDateFormatter alloc] init];
```

Comment:
Probably need to set a locale here to resolve the issue I commented on the PR about.